docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,616 @@ license: |
+      <td>any, boolean</td>
+      <td>Returns the first value of expression for a group of rows. If <code>isIgnoreNull</code> is true, returns only non-null values, default is false. This function is non-deterministic.</td>
+    </tr>      
+    <tr>
+      <td><b>kurtosis</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the kurtosis value calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>{last | last_value}</b>(<i>expression[, isIgnoreNull]</i>)</td>
+      <td>any, boolean</td>
+      <td>Returns the last value of expression for a group of rows. If <code>isIgnoreNull</code> is true, returns only non-null values, default is false. This function is non-deterministic.</td>
+    </tr>      
+    <tr>
+      <td><b>max</b>(<i>expression</i>)</td>
+      <td>any numeric, string, date/time or arrays of these types</td>
+      <td>Returns the maximum value of the expression.</td>
+    </tr>          
+    <tr>
+      <td><b>max_by</b>(<i>expression1, expression2</i>)</td>
+      <td>any numeric, string, date/time or arrays of these types</td>
+      <td>Returns the value of expression1 associated with the maximum value of expression2.</td>
+    </tr>   
+    <tr>
+      <td><b>min</b>(<i>expression</i>)</td>
+      <td>any numeric, string, date/time or arrays of these types</td>
+      <td>Returns the minimum value of the expression.</td>
+    </tr>          
+    <tr>
+      <td><b>min_by</b>(<i>expression1, expression2</i>)</td>
+      <td>any numeric, string, date/time or arrays of these types</td>
+      <td>Returns the value of expression1 associated with the minimum value of expression2.</td>
+    </tr>      
+    <tr>
+      <td><b>percentile</b>(<i>expression, percentage [, frequency]</i>)</td>
+      <td>numeric Type, double, integral type</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the exact percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>percentile</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>numeric type; double; integral type</td>
+      <td>Percentage array is an array of number between 0 and 1; frequency is a positive integer. Returns the exact percentile value array of numeric expression at the given percentage(s).</td>
+    </tr>        
+    <tr>
+      <td><b>{percentile_approx | percentile_approx}</b>(<i>expression, percentage [, frequency]</i>)</td>
+      <td>numeric, date, timestamp; double; integral</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the approximate percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>{percentile_approx | percentile_approx}</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>numeric, date, timestamp; double; integral</td>
```

Comment:
What does `;` mean?

Reply:
I try to use `;` to separate each argument's data type. Will it be better to use this way? `numeric|date|timestamp, double, integral`

Reply:
Ah, I see. Rather than using `;`, could we separate it into two entries?
https://github.com/apache/spark/pull/28120#discussion_r404541183

Reply:
ok